docs/en/stack/ml/anomaly-detection/anomaly-detection-scale.asciidoc

```diff
@@ -0,0 +1,228 @@
+{anomaly-jobs} model memory API] manually. The estimation is based on the 
+analysis configuration details for the job and cardinality estimates for the 
+fields it references.
+
+
+[[job-config]]
+== Job configuration guidelines
+
+
+[[dedicated-results-index]]
+=== Using a dedicated results index
+
+If a {anomaly-job} generates a lot of data, use a dedicated index to store the 
+results. This way, you can retain results in a shorter time. If you use a shared 
+index, the different {anomaly-jobs} may slow down.
+
+When you want to delete a job, the delete job API instantly drops the index 
+containing the job’s results if you use a dedicated index. If you use a shared 
+one, however, it may take hours to delete all the individual documents related 
+to the deleted job.
+
+
+[[search-optimization]]
+=== Optimizing search queries
+
+There are different ways to write {es} queries and some of them are more 
+efficient than others. If you are operating on a big scale, make sure that your 
+{dfeed} query is as efficient as possible. Consult 
+{ref}/tune-for-search-speed.html[Tune for search speed] to learn more about {es} 
+performance tuning.
+
+
+[[influencers-detectors]]
+=== Choosing influencer fields and detectors
+
+Optimize your {anomaly-job} by choosing only relevant influencer fields and 
+detectors.
+
+
+[[short-bucket-span]]
+=== Avoiding short bucket span for large jobs
+
+If you have a large amount of data, use longer bucket span. If your data 
+requires short bucket span, split the data to avoid having high cardinality. The 
+combination of short bucket span and high cardinality is a resource-intensive 
+situation that you may want to avoid.
+
+
+[[population-analysis]]
+=== Using population analysis
+
+Population analysis is more memory efficient than individual analysis of each 
+series. It builds a profile of what a "typical" entity does over a specified 
+time period and then identifies when one is behaving abnormally compared to the 
+population. Use population analysis for analyzing high cardinality fields if you 
+expect that the entities of the population generally behave in the same way.
+
+
+[[pre-aggregating-data]]
+=== Pre-aggregating data
+
+{anomaly-jobs-cap} always work on summary statistics calculated for the buckets. 
+The statistics can be calculated in the job itself or via aggregations. It is 
+more efficient to use an aggregation when it's possible, as in this case, the 
+data node does the heavy-lifting instead of the {ml} node.
+
+In certain cases, you cannot do aggregations to increase performance. For 
+example, categorization jobs use the full log message to detect anomalies, so 
+this data cannot be aggregated. If you have many influencer fields, it may not 
+be beneficial to use an aggregation either, because only a few documents in each 
+bucket may have the combination of all the different influencer fields.
+
+Consult <<ml-configuring-aggregation>> to learn more.
```

Comment:
`Please see` or `See` sounds friendlier than `Consult`

Reply:
We don't usually use `see` because of accessibility reasons. I added `please` before `consult`.